pkg/asset/cluster/tfvars.go

```diff
@@ -270,6 +272,26 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		if err != nil {
 			return err
 		}
+
+		baseImage := string(*rhcosImage)
+		var baseImageURL string
+
+		// If baseImage is empty then it was not provided by the user and we need to create it.
```

Comment:
I don't think we need to leave the possibility to provide a base image, the installer pins the image for a reason.

Reply:
This is an OCP requirement to give the possibility to provide a base image for the installer by setting OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE variable. Also this is good for CI, where we don't need to create new images every time, since we can reuse the existing one.

Reply:
Fair enough.